.circleci/config.yml

```diff
@@ -0,0 +1,93 @@
+alias:
+  - &install_dzil 
+    run:
+      command: |
+        cpm install -g --no-test Dist::Zilla Dist::Zilla::App::Command::cover ExtUtils::MakeMaker
+      name: Install Dzil
+  - &install_author_deps
+    run: 
+      name: Install dzil author dependencies
+      command: |
+        # TODO remove this line before merge
+        cpanm git@github.com:chylli-binary/perl-Dist-Zilla-PluginBundle-Author-DERIV.git@bumpversion
```

Comment:
Please remove this line

Reply:
thanks. will remove before merge, still need it for test now.